52_wa_diff_shipment.py

```diff
@@ -0,0 +1,263 @@
+
+other_states_shipment
+
+other_states_shipment_copy = other_states_shipment.copy()
+
+# other_states_shipment_copy["shipment_per_100k"] = (other_states_shipment_copy["QUANTITY"] / other_states_shipment_copy["Population"]) * 100_000
+other_states_shipment_copy["shipment_per_100k"] = (
+    (
+        other_states_shipment_copy["dos_str"]
+        * other_states_shipment_copy["DOSAGE_UNIT"]
+        * other_states_shipment_copy["MME_Conversion_Factor"]
+    )
+    / (other_states_shipment_copy["Population"])
+    * 100000
+)
+
+other_states_shipment_copy
+
+# calculating and displaying results for drug related deaths - year, state and county wise
+
+other_states_result = (
+    other_states_shipment_copy.groupby(["Year", "StateName", "county_test"])[
+        "shipment_per_100k"
+    ]
+    .sum()
+    .reset_index()
+)
+other_states_result.head()
+
+washington_summary = pd.DataFrame(
+    washington_prescriptions_result.describe()["shipment_per_100k"]
+).rename(columns={"shipment_per_100k": "Opioid Shipment per 100k Residents - Texas"})
+other_states_summary = pd.DataFrame(
+    other_states_result.describe()["shipment_per_100k"]
+).rename(
+    columns={"shipment_per_100k": "Opioid Shipment per 100k Residents - Comp States"}
+)
+stats = pd.concat([washington_summary, other_states_summary], axis=1)
+stats
+# calculating yearwise drug related deaths results for Oregon
+Oregon_results = (
+    other_states_shipment_copy[other_states_shipment_copy["StateName"] == "Oregon"]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
+Oregon_results
+# calculating yearwise drug related deaths results for Idaho
+Idaho_results = (
+    other_states_shipment_copy[other_states_shipment_copy["StateName"] == "Idaho"]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
+Idaho_results
+# calculating yearwise drug related deaths results for Montana
+montana_results = (
+    other_states_shipment_copy[other_states_shipment_copy["StateName"] == "Montana"]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
+montana_results
+# calculating yearwise drug related deaths results for Nevada
+nevada_results = (
+    other_states_shipment_copy[other_states_shipment_copy["StateName"] == "Nevada"]
+    .groupby("Year")["shipment_per_100k"]
+    .mean()
+)
+nevada_results
+
+wa_result = (
+    washington_prescriptions_result.groupby("Year")["shipment_per_100k"]
+    .mean()
+    .reset_index()
+)
+comp_result = (
+    other_states_result.groupby(["Year"])["shipment_per_100k"].mean().reset_index()
+)
+comp_result
+wa_result
+
+florida_summary = pd.DataFrame(wa_result.describe()["shipment_per_100k"]).rename(
+    columns={"shipment_per_100k": "Opioid Shipment per 100k Residents - Washington"}
+)
```

Comment:
You would want to replace the name to washington_summary, since we are looking at the statistics regarding opioid shipment in Washington.

Reply:
Great! Attention to detail matters and goes a long way.